turbine/src/cluster_nodes.rs

```diff
@@ -393,44 +407,90 @@ fn cmp_nodes_stake(a: &Node, b: &Node) -> Ordering {
 
 // Dedups socket addresses so that if there are 2 nodes in the cluster with the
 // same TVU socket-addr, we only send shreds to one of them.
+// This will prefer to keep the ports with the latest outset value, if possible
 // Additionally limits number of nodes at the same IP address to
 // MAX_NUM_NODES_PER_IP_ADDRESS.
-fn dedup_tvu_addrs(nodes: &mut Vec<Node>) {
+fn dedup_tvu_addrs(nodes: &mut Vec<Node>, keep_identity: Pubkey) {
```

Comment:
I'm leaning towards just getting rid of the `keep_identity` stuff now. There's no known issue where we would expect to purge our own identity now, right?

Reply:
emphasis on "known". And the cost of keeping this logic is very low.

Reply:
changes intended for backport must be the smallest viable